docs/cni/kubernetes/KubernetesIntegration.md

```diff
@@ -41,37 +45,114 @@ sudo chmod +x calicoctl
 sudo ETCD_AUTHORITY=<ETCD_IP>:<ETCD_PORT> ./calicoctl node
 ```
 
-#### 2. Configure the Network Plugin 
-The Calico network plugin for Kubernetes uses the `calico_kubernetes.ini` file to read in user configuration.
+See the [`calicoctl node` documentation](https://github.com/projectcalico/calico-containers/blob/master/docs/calicoctl/node.md#calicoctl-node)
+for more information.
+
+#### Example systemd unit file (calico-node.service)
+```
+[Unit]
+Description=calicoctl node
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=root
+Environment=ETCD_AUTHORITY=<ETCD_IP>:<ETCD_PORT>
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/wget -N -P /opt/bin http://www.projectcalico.org/builds/calicoctl
```

Comment:
This will always install the latest calico, right?  Might want to note that (as it could unexpected break things on reboot, for example)